app/views/vendors/show.html.erb

```diff
@@ -56,11 +60,24 @@
    <form>
      <div class="row collapse">
        <h3>Add Review:</h3>
-
-       <div class="large-10 small-8 columns">
-         <input type="text"/>
-       </div>
-
+       <%= form_for [@vendor, @review] do |f| %>
+       <%= f.label :radio_button, "Rating" %>
+       <%= f.radio_button(:rating, "1") %>
+       <%= f.label(:rating_1, "1") %>
+       <%= f.radio_button(:rating, "2") %>
+       <%= f.label(:rating_2, "2") %>
+       <%= f.radio_button(:rating, "3") %>
+       <%= f.label(:rating_3, "3") %>
+       <%= f.radio_button(:rating, "4") %>
+       <%= f.label(:rating_4, "4") %>
+       <%= f.radio_button(:rating, "5") %>
+       <%= f.label(:rating_5, "5") %>
+
+       <%= f.label :body, "Comments:" %>
+       <%= f.text_field :body %>
+
+       <%= f.submit %>
+       <% end %>
```

Comment:
There are a couple places in this file (outside just these diffs shown here) with messy indentation - try to clean that up some!